sdk/aqueduct/integrations/mongodb_integration.py

```diff
@@ -0,0 +1,171 @@
+import json
+from typing import Any, Dict, List, Optional
+
+from aqueduct.artifacts import utils as artifact_utils
+from aqueduct.artifacts.base_artifact import BaseArtifact
+from aqueduct.artifacts.metadata import ArtifactMetadata
+from aqueduct.artifacts.table_artifact import TableArtifact
+from aqueduct.dag import DAG
+from aqueduct.dag_deltas import AddOrReplaceOperatorDelta, apply_deltas_to_dag
+from aqueduct.enums import ArtifactType, ExecutionMode, LoadUpdateMode
+from aqueduct.error import InvalidUserArgumentException
+from aqueduct.integrations.integration import Integration, IntegrationInfo
+from aqueduct.integrations.sql_integration import find_parameter_artifacts, find_parameter_names
+from aqueduct.operators import (
+    ExtractSpec,
+    MongoExtractParams,
+    Operator,
+    OperatorSpec,
+    RelationalDBLoadParams,
+    SaveConfig,
+)
+from aqueduct.utils import artifact_name_from_op_name, generate_uuid
+
+from aqueduct import globals
+
+
+class MongoDbCollectionIntegration(Integration):
```

Comment:
naming nit: Can we capitalize "DB" throughout?

Reply:
Actually I found we are using `Db` for all variable / class names throughout our codebase:

<img width="737" alt="Screen Shot 2022-11-03 at 11 39 23 AM" src="https://user-images.githubusercontent.com/10411887/199807122-19089dfc-145f-4519-b036-9d3ab73b30f0.png">

Reply:
yes but I think the canonical way to style it is `DB`, similar to how we should use `ID` and not `Id` (which I don't think is consistent in our codebase). Or do you want to hold off on the rename and do it all at one?

Reply:
based on https://learn.microsoft.com/en-us/previous-versions/dotnet/netframework-1.1/141e06ef(v=vs.71)?redirectedfrom=MSDN , `DB` is the way to go so I will move all mongo code to use `DB`. Let's migrate other ones separately.